src/staging/project_setup.py

```diff
@@ -95,8 +95,8 @@ def generate_project_name(
 ) -> str:
     res = {
         ProjectType.DEVEL: "devel:BCI:",
-        ProjectType.CR: f"home:{osc_username}:BCI:CR:",
-        ProjectType.STAGING: f"home:{osc_username}:BCI:Staging:",
+        ProjectType.CR: f"devel:BCI:CR:",
+        ProjectType.STAGING: f"devel:BCI:Staging:",
```

Comment:
There's a catch with this change: you must be a maintainer of devel:BCI to run the bot locally. Previously, it would create a staging project in your home project, which everyone owns.

Then again, the home: scheduler is de-prioritized first.

Reply:
I never figured out how to run the bot locally. how would one do that?

Reply:
in general I think moving out of "home:defolos" is a good improvement. having staging below devel:BCI means it will inherit signing keys and permissions from the parent project, which might not be a good idea. 

I wonder if we can request a "devel:BCI-CI" and then move staging and CR there.

Reply:
Can we have a subproject, like `devel:BCI:CI`, that does not inherit certain things? Or is that only possible with a new devel project?

Reply:
> Can we have a subproject, like `devel:BCI:CI`, that does not inherit certain things? Or is that only possible with a new devel project?

only possible with another project hierarchy